week-4/implement/repeat.test.js

```diff
@@ -23,3 +25,24 @@
 // Given a target string str and a negative integer count,
 // When the repeat function is called with these inputs,
 // Then it should throw an error or return an appropriate error message, as negative counts are not valid.
+
+function repeatString(str, num) {
+  if (num < 0) {
+    return "negative count is not valid";
+  }
+  let repeatedString = str.repeat(num);
+  if (num === 0) {
+    return "";
+  }
+  if (num === 1) {
+    return str;
+  } else return repeatedString;
```

Comment:
Check what happens when you give `str.repeat(..)` the number 0 or 1. You might find that you can remove a few `if` statements from your function 😉